.github/workflows/application-signals-python-e2e-ec2-test.yml

```diff
@@ -32,11 +32,12 @@ env:
   METRIC_NAMESPACE: AppSignals
   LOG_GROUP_NAME: /aws/appsignals/generic
   ADOT_WHEEL_NAME: ${{ inputs.staging_wheel_name }}
-  TEST_RESOURCES_FOLDER: /home/runner/work/aws-application-signals-test-framework/aws-application-signals-test-framework
```

Comment:
Out of curiosity, why is this not needed anymore?

Reply:
Since this is the adot python repo, we did full github checkout for the `aws-application-signals-test-framework` and use the codes there, so don't need to use the full path. I will need to add it back when moving E2E tests to test-framework repo.